parsec/api/data/sequester.py

```diff
@@ -0,0 +1,39 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+from enum import Enum
+from typing import Dict, Any
+import attr
+from parsec.api.data.base import BaseAPIData
+from parsec.serde import fields, post_load
+from parsec.serde.schema import BaseSchema
+from pendulum import DateTime
+
+
+class EncryptionKeyFormat(Enum):
+    RSA = "RSA"
+
+
+EncryptionKeyFormatFields = fields.enum_field_factory(EncryptionKeyFormat)
+
+
+@attr.s(slots=True, frozen=True, auto_attribs=True, kw_only=True, eq=False)
+class SequesterServiceEncryptionKey(BaseAPIData):
+    class SCHEMA_CLS(BaseSchema):
+
+        type = fields.CheckedConstant("sequester_service_encryption_key", required=True)
+        encryption_key = fields.Bytes(required=True)
+        encryption_key_format = EncryptionKeyFormatFields(
+            required=True
+        )  # TODO: does it has to be signed
```

Comment:
Not sure I understand this comment 🤔 

The rule of thumb in crypto is often "don't try to be smart" ^^
On top of that having this field not signed require to have two separate schemas (one signed and one containing the signed one along with this field) so it's just more complexity for no added benefit. So we should definitely keep this field signed 😄

Reply:
Removed